lampo-common/src/keys.rs

```diff
@@ -113,9 +115,9 @@ impl NodeSigner for LampoKeysManager {
     fn ecdh(
         &self,
         recipient: lightning::sign::Recipient,
-        other_key: &bitcoin::secp256k1::PublicKey,
-        tweak: Option<&bitcoin::secp256k1::Scalar>,
-    ) -> Result<bitcoin::secp256k1::ecdh::SharedSecret, ()> {
+        other_key: &lightning::bitcoin::secp256k1::PublicKey,
+        tweak: Option<&lightning::bitcoin::secp256k1::Scalar>,
+    ) -> Result<lightning::bitcoin::secp256k1::ecdh::SharedSecret, ()> {
```

Comment:
same here we do not referer in this crate with `lightning`, it is wrong.

Please see how we use it in other part of the code

Reply:
Fixed